crates/sui-adapter/src/adapter.rs

```diff
@@ -554,7 +554,8 @@ fn process_successful_execution<S: Storage + ParentSync>(
         obj.data
             .try_as_move_mut()
             .expect("We previously checked that mutable ref inputs are Move objects")
-            .update_contents_and_increment_version(new_contents);
+            .update_contents_and_increment_version(new_contents)
```

Comment:
it's better to check the error is indeed `SuiError:ObjectTooBig` then we add more errors in `new_from_execution`, we will likely forget to update there too